rrfs-test/validated_yamls/templates/obtype_config/abi_g16.yaml

```diff
@@ -0,0 +1,481 @@
+         SurfaceWindGeoVars: uv
+         obs options:
+           Sensor_ID:  abi_g16 
+           EndianType: little_endian
+           CoefficientPath: data/crtm/
+           IRVISlandCoeff: IGBP
+         linear obs operator:
+           Absorbers: [H2O, O3]
+  # Observation Bias Correction (VarBC)
+  # -----------------------------------
+       obs bias:
+         input file: data/obs/abi_g16.satbias.nc4
+         output file: out_abi_g16.satbias.nc4
+         variational bc:
+           predictors:
+           - name: constant
+           - name: lapseRate
+             order: 2
+             tlapse: &abi_g16_tlapse data/obs/abi_g16.tlapse.txt
+           - name: lapseRate
+             tlapse: *abi_g16_tlapse
+           - name: emissivityJacobian
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 4
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 3
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 2
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+         covariance:
+           minimal required obs number: 20
+           variance range: [1.0e-6, 10.0]
+           step size: 1.0e-4
+           largest analysis variance: 10000.0
+           prior:
+             input file: data/obs/abi_g16.satbias_cov.nc4
+             inflation:
+               ratio: 1.1
+               ratio for small dataset: 2.0
+           output file: out_abi_g16.satbias_cov.nc4
+  # Observation Filters (QC)
+  # ------------------------
+       obs prior filters:
+       - filter: Perform Action
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g16_channels
+         action:
+           name: assign error
+           error parameter vector: [ 2.2, 1.756, 1.587, 1.185, 2.2, 2.2, 2.2, 2.2, 2.2, 2.2 ]
+
+       - filter: Variable Assignment
+         assignments:
+         - name: DerivedMetaData/surfaceFlag
+           type: int
+           function:
+             name: IntObsFunction/Conditional
+             options:
+               defaultvalue: 4
+               firstmatchingcase: true
+               cases:
+               - where:
+                 - variable:
+                     name: GeoVaLs/water_area_fraction
+                   minvalue: 0.99
+                 value: 0
+               - where:
+                 - variable:
+                     name: GeoVaLs/land_area_fraction
+                   minvalue: 0.99
+                 value: 1
+               - where:
+                 - variable:
+                     name: GeoVaLs/ice_area_fraction
+                   minvalue: 0.99
+                 value: 2
+               - where:
+                 - variable:
+                     name: GeoVaLs/surface_snow_area_fraction
+                   minvalue: 0.99
+                 value: 3
+       - filter: Variable Assignment
```

Comment:
Is this filter of the same nature as the previous? I would prefer to add a space and a comment.